templates/includes/sidebar.html

```diff
@@ -1,4 +1,8 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <a href="{% url "taxi:driver-detail" pk=user.id %}">{{ user.username }}</a>
```

Comment:
Ensure that the `driver-detail` URL pattern correctly expects a `pk` parameter that matches `user.id`. Mismatches could lead to `NoReverseMatch` errors. Verify that the URL pattern is correctly defined in your `urls.py`.